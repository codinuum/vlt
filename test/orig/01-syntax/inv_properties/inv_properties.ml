let () = [%LOG [%DEBUG] "msg" [%PROPERTIES 0]]
