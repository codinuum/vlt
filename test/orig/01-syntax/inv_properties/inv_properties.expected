File "inv_properties.ml", line 1, characters 43-44:
1 | let () = [%LOG [%DEBUG] "msg" [%PROPERTIES 0]]
                                               ^
Error: This expression has type int but an expression was expected of type
         (string * string) list
